qt/scientific_interfaces/ISISReflectometry/GUI/Preview/PreviewPresenter.cpp

```diff
@@ -74,16 +74,17 @@ void PreviewPresenter::notifyAutoreductionPaused() { updateWidgetEnabledState();
 
 void PreviewPresenter::updateWidgetEnabledState() {
   if (m_mainPresenter->isProcessing() || m_mainPresenter->isAutoreducing()) {
-    m_view->disableApplyButton();
+    m_view->disableMainWidget();
   } else {
-    m_view->enableApplyButton();
+    m_view->enableMainWidget();
   }
 }
 
 /** Notification received when the user has requested to load a workspace. If it already exists in the ADS
  * then we use that and continue to plot it; otherwise we start an async load.
  */
 void PreviewPresenter::notifyLoadWorkspaceRequested() {
+  m_view->disableMainWidget();
   auto const name = m_view->getWorkspaceName();
   try {
```

Comment:
If I try to load `POLREF37125` (which is not a matrix workspace and therefore throws an error) the tab stays disabled and I can't attempt to load a different run. I'm not sure if it's anything to do with not re-enabling the view as part of this try/catch, or if the issue is elsewhere.

Similarly, when testing with `INTER45455_inst`, if `ReflectometryISISSumBanks` throws an error then the tab doesn't seem to be re-enabled.